src/vcpkg/paragraphs.cpp

```diff
@@ -549,20 +537,16 @@ namespace vcpkg::Paragraphs
             auto maybe_port_location = (*port_entry)->get_version(*baseline_version);
             const auto port_location = maybe_port_location.get();
```

Comment:
This is a pointer so there's no benefit to be had for moving it. Also, `try_load_port_required` accepts by `const&` because the data in question is normally owned by the registry...